### ISSUE TITLE
[A11y Maintenance Audit]: VA Benefit Letters

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#45635]
## Auditor(s)

Josh Kim @joshkimux

## Automated

- [x] Run axe checks on every page or unique state including responsive (required)
- [x] WAVE on starting page (unauthenticated)

## Manual

- [x] Color contrast checks (recommend using whocanuse.com for beyond compliance reports)
- [x] Color blindness checks
- [x] Zoom layouts to 400% and inspect them for readability. If layouts break at 400%, I will start reducing them until they become stable, and log the zoom ratio when things started breaking.
- [x] Keyboard navigation

### This audit will focus on the following combinations
- [ ] Edge and NVDA
- [x] Safari and iOS Voice Over
- [x] Safari and iOS Macbook

### This audit will not cover the following screen readers and browsers
- [ ] Talkback
- [ ] JAWs
- [ ] Firefox (doesn't work on staging)

## Staging or Production Link

[Download VA Letters (Staging)](https://staging.va.gov/records/download-va-letters/)

## Prototype Link

[Prototypes, test users, and more in read me](https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Products/Records/VA%20Documents%20and%20Letters/README.md)

## Test Users

- [ ] vets.gov.user+299@gmail.com

**Comment by joshkimux** [comment]
@skylerschain for your awareness!
